projects/ui-components/lf-login/login-utils/lf-login.service.ts

```diff
@@ -127,14 +128,10 @@ export class LfLoginService {
   }
 
   /** @internal */
-  async parseTokenResponseAsync(response: GetAccessTokenResponse): Promise<string | undefined> {
+  async parseTokenResponseAsync(response: GetAccessTokenResponse): Promise<AuthorizationCredentials | undefined> {
     try {
       const authorizationCredentials: AuthorizationCredentials = this.getExchangeCodeSuccessResponse(response);
```

Comment:
is there a case where this doesn't emit the loggedIn event afterwards?

Reply:
there are two places that called parseTokenResponseAsync: `refreshTokenAsync` and `exchangeCodeForTokenAsync`, and we emit loggedInComplete afterwards at both places